### PR TITLE
SL-614: use mother's address when registering a child

### PR DESCRIPTION
**Pull request by cioan**
Finally, got all mother's requested info into the child registration page:
Screenshot 2024-05-31 at 10 38 15 AM

### CODE REVIEW

File: omod/src/main/java/org/openmrs/module/pihcore/page/controller/children/FindChildrenPageController.java [hunk 1]
```diff
@@ -41,14 +44,23 @@ public void controller(PageModel model, UiUtils ui, UiSessionContext uiSessionCo
...
         // pass the mother's info to the baby registration form as a list of property-value in the following format:
         // SECTION.QUESTION.FIELD , this matches the existing format of the registration form
         SimpleObject initialValues = SimpleObject.create(
                 "demographics.mothersFirstNameLabel.mothersFirstName", patient.getGivenName(),
-                "contactInfo.phoneNumberLabel.phoneNumber", phoneNumber
+                "contactInfo.personAddressQuestion.country", patient.getPersonAddress().getCountry(),
+                "contactInfo.phoneNumberLabel.phoneNumber", phoneNumber,
+                "registerRelationships.relationships_mother.relationship_type", motherToChildRelationshipType.getUuid() + "-A",
+                "registerRelationships.relationships_mother.other_person_uuid", patient.getUuid(),
+                "registerRelationships.relationships_mother.other_person_name", patientInfo,
```
Comment (mogoodrich) [inline omod/src/main/java/org/openmrs/module/pihcore/page/controller/children/FindChildrenPageController.java:60]: These are just display field we need to pre-set with data, I'm guessing?

**Review by mogoodrich — APPROVED** [review]
Thanks @cioan , LGTM!

File: omod/src/main/java/org/openmrs/module/pihcore/page/controller/children/FindChildrenPageController.java (hunk 1 above)
Comment (cioan) [reply omod/src/main/java/org/openmrs/module/pihcore/page/controller/children/FindChildrenPageController.java:60]: Yes, that's correct.

**Comment by cioan** [comment]
Thanks @mogoodrich !

**Pull request merged**